src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -2,32 +2,124 @@
 
 import java.util.List;
 
-public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+class Node<T> {
+    T data;
+    Node<T> prev;
+    Node<T> next;
+
+    Node(T data) {
+        this.data = data;
+        this.prev = null;
+        this.next = null;
+    }
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor to initialize them directly.